test/integration/kubernetes.test.ts

```diff
@@ -226,6 +244,59 @@ tap.test('snyk-monitor pulls images from a private ECR and sends data to kuberne
     'snyk-monitor sent expected data to upstream in the expected timeframe');
 });
 
+tap.test('snyk-monitor pulls images from a local registry and sends data to kubernetes-upstream', async (t) => {
+  t.teardown(async () => {
+    console.log('Begin removing local container registry...');
+    await setup.removeLocalContainerRegistry();
+    console.log('Removed local container registry');
+    console.log('Begin removing "kind" network...');
+    await setup.removeUnusedKindNetwork();
+    console.log('Removed "kind" network');
+  });
+  
+  if (process.env['TEST_PLATFORM'] !== 'kind') {
+    t.pass('Not testing local container registry because we\'re not running in KinD');
+    return;
+  }
+  t.plan(4);
+
+  const deploymentName = 'python-local';
+  const namespace = 'services';
```

Comment:
Does being in `services` not affect the count of workloads we are expecting in a previous test? (Sorry if we're repeating the conversation, can't remember what we found)

Reply:
No worries, I didn't quite remember myself!! It's ok as this deployment is added later and what was causing failures, it was actually the job that was causing failures so I removed the namespace from the job.